### PR TITLE
Designed Win Page 

### PR DESCRIPTION
**Pull request by amitojsingh**
Designed win page. #7 
/win is the route
Screenshot 2023-12-15 at 1 05 35 PM

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -44,19 +45,20 @@ function App() {
   const router = createBrowserRouter(
     createRoutesFromElements(
       <Route path={PAGES.ROOT} element={<RootLayout />}>
-        <Route path='' element={<Home />} />
+        <Route path="" element={<Home />} />
```
Comment (MySmilingTurban) [inline src/App.tsx:48]: change to single quotes

File: src/App.tsx [hunk 2]
```diff
@@ -44,19 +45,20 @@ function App() {
...
+        <Route path={PAGES.WIN} element={<Win />} />
         <Route path={PAGES.WINCOIN} element={<WinCoin />} />
         <Route path={PAGES.WORDS} element={<WordsPageLayout />}>
           <Route path={PAGES.DEFINITION} element={<Defintion />} />
           <Route path={PAGES.EXAMPLES} element={<Examples />} />
           <Route path={PAGES.SEMANTICS} element={<Semantics />} />
           <Route path={PAGES.INFORMATION} element={<Information />} />
         </Route>
-        <Route path='*' element={<NotFound />} />
+        <Route path="*" element={<NotFound />} />
```
Comment (MySmilingTurban) [inline src/App.tsx:61]: change to single quotes

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (amitojsingh) [reply src/App.tsx]: fixed veer ji

File: src/App.tsx (hunk 2 above)
Comment (amitojsingh) [reply src/App.tsx]: fixed veer ji

**Review by MySmilingTurban — APPROVED** [review]
looks good to me!

**Pull request merged**